x-pack/plugins/global_search_providers/server/providers/saved_objects/provider.test.ts

```diff
@@ -195,18 +195,22 @@ describe('savedObjectsResultProvider', () => {
     const results = await provider.find({ term: 'term' }, defaultOption, context).toPromise();
     expect(results).toEqual([
       {
+        icon: undefined,
```

Comment:
@pgayvallet Are these changes ok?

Reply:
I think you don't need the `icon: undefined` (it was only displayed in the error message because the `meta` attribute was also missing, but undefined properties are ignored from the actual diff I believe)